### DISCUSSION TITLE
Whisper training code

### DISCUSSION THREAD

**Discussion by jayakrishnanmm** [openai/whisper#1879] [General]
Looks  like the  whisper github  don't have a training script. I checked  the fine tuning script. Can I use it for training  a  Whisper model  from  scratch , with my own data ?

**Comment by afsara-ben** [comment]
same q

**Comment by toanhuynhnguyen** [comment]
Try this, https://www.youtube.com/watch?v=O60EnXcbi6g 
And its demo code here: https://github.com/vb100/whisper_ai_finetune
